### PR TITLE
Ddt 1227 booking summaries

### PR DESCRIPTION
**Pull request by pimg**
Resolves: DDT-1227

### CODE REVIEW

File: edocumentation-service/src/main/java/org/dcsa/edocumentation/service/mapping/BookingSummaryMapper.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    }
+
+    return switch (modeOfTransportCode) {
+      case "1" -> DCSATransportType.VESSEL;
+      case "2" -> DCSATransportType.RAIL;
+      case "3" -> DCSATransportType.TRUCK;
+      case "8" -> DCSATransportType.BARGE;
+      default -> throw ConcreteRequestErrorMessageException.internalServerError("Could not convert TransportTypeCode to DCSA TransportType");
+
+    };
```
Comment (nt-as) [inline edocumentation-service/src/main/java/org/dcsa/edocumentation/service/mapping/BookingSummaryMapper.java:34]: These values are in the database, so we could just fetch them as a separate entity (which makes it easier to support new values as they are added and we can use our standard processes for the value).

Comment (nt-as) [reply edocumentation-service/src/main/java/org/dcsa/edocumentation/service/mapping/BookingSummaryMapper.java:34]: (fetch them via a JOIN)

**Review by douradofrazer — CHANGES_REQUESTED** [review]

File: .github/workflows/master.yml [hunk 2]
```diff
@@ -33,6 +33,12 @@ jobs:
       - name: Build Edocumentation
         run: cd DCSA-Edocumentation && mvn clean -U -B package -Ddcsa.artifacttype=-SNAPSHOT
 
+      - name: Run Integration Tests
+        run: |
+          echo "testcontainers.reuse.enable=true" > ~/.testcontainers.properties
```
[the remark this answers is not in the document]
Comment (pimg) [reply .github/workflows/master.yml:38]: this is a required attribute in order to make test containers reusable. The .withReuse(true) is not sufficient

File: edocumentation-application/src/main/resources/application.yml [hunk 3]
```diff
@@ -35,7 +37,7 @@ spring:
 
 server:
   port: 9090
-  servlet.context-path: /v2
+  servlet.context-path: /
```
[the remark this answers is not in the document]
Comment (pimg) [reply edocumentation-application/src/main/resources/application.yml]: fixed removed property

File: edocumentation-service/src/main/java/org/dcsa/edocumentation/service/mapping/BookingSummaryMapper.java (hunk 1 above)
Comment (pimg) [reply edocumentation-service/src/main/java/org/dcsa/edocumentation/service/mapping/BookingSummaryMapper.java]: fixed

**Review by douradofrazer — APPROVED** [review]

**Pull request merged**